libcontainerd/local/local_windows.go

```diff
@@ -313,8 +313,8 @@ func (c *client) createWindows(id string, spec *specs.Spec, runtimeOptions inter
 		}
 		for _, d := range spec.Windows.Devices {
 			// Per https://github.com/microsoft/hcsshim/blob/v0.9.2/internal/uvm/virtual_device.go#L17-L18,
-			// this represents an Interface Class GUID.
-			if d.IDType != "class" {
+			// these represent an Interface Class GUID.
+			if d.IDType != "class" && d.IDType != "vpci-class-guid" {
```

Comment:
Yeah, I was thinking about doing exactly that earlier today. There's hcsshim/pkg/annotations, so something similar for these constants makes sense to me. I have a few (too many) hcsshim PRs in-flight though right now. -_- But this one's on my radar.

Reply:
Ah, yes, I know you have some other ones. Let's see if we can chat with some of the MS maintainers to get some moving.

BTW; are you on any of the docker or CNCF slack channels?

Reply:
I'm not on any of the Slack channels, as it happens.

Reply:
Ah, that's a pity, thought it would be easier to (when needed) have a synchronous chat (also with the containerd maintainers, who mostly reside on the CNCF slack). If you ever decide to join that slack, let me know, then I can connect you with them as well 👍